typescript/test/testdata/texttospeech/src/v1/text_to_speech_client.ts.baseline

```diff
@@ -232,9 +232,20 @@ export class TextToSpeechClient {
   // -------------------
   // -- Service calls --
   // -------------------
-  /*
-  Returns a list of Voice supported for synthesis. 
-  */
+/**
+ *  Returns a list of Voice supported for synthesis.
+ * 
+ * @param { TYPE_STRING } request.language_code 
```

Comment:
Looks like we need to have a mapping from `TYPE_*` to actual JavaScript types.

Reply:
Got it!